lib/puppet_metadata/operatingsystem.rb

```diff
@@ -110,10 +110,10 @@ class OperatingSystem
         '4' => '2012-02-29',
         '3' => '2010-10-31',
       },
-      # https://endoflife.date/rocky-linux
+      # https://wiki.rockylinux.org/rocky/version/
       'Rocky' => {
-        '9' => nil,
-        '8' => '2029-05-31',
+        '9' => '2027-05-31',
+        '8' => '2024-05-31',
```

Comment:
I'd keep the actual EOL date, not the active support phase:
```suggestion
        '9' => '2032-05-31',
        '8' => '2029-05-31',
```